client/src/modules/payroll/rubrics/modals/rubric.modal.html

```diff
@@ -0,0 +1,172 @@
+<form name="RubricForm" bh-submit="RubricModalCtrl.submit(RubricForm)" novalidate>
+  <div class="modal-header">
+  <ol class="headercrumb">
+    <li ng-if="RubricModalCtrl.isCreating" class="title">
+      <span translate>PAYROLL_RUBRIC.ADD_PAYROLL_RUBRIC</span>
+      <label class="badge badge-success" translate>FORM.LABELS.CREATE</label>
+    </li>
+    <li ng-if="!RubricModalCtrl.isCreating" class="title">
+      <span translate>PAYROLL_RUBRIC.EDIT_PAYROLL_RUBRIC</span>
+      <label class="badge badge-success" translate>FORM.LABELS.UPDATE</label>
+    </li>
+  </ol>
+</div>
+
+  <div class="modal-body">
+    <div class="form-group" ng-class="{ 'has-error' : RubricForm.$submitted && RubricForm.label.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.DESIGNATION</label>
+      <input name="label" ng-model="RubricModalCtrl.rubric.label" autocomplete="off" class="form-control" required>
+      <div class="help-block" ng-messages="RubricForm.label.$error" ng-show="RubricForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error' : RubricForm.$submitted && RubricForm.label.$invalid }">
+      <label class="control-label" translate>FORM.LABELS.ABBREVIATION</label>
+      <input name="abbr" ng-model="RubricModalCtrl.rubric.abbr" autocomplete="off" class="form-control" required>
+      <div class="help-block" ng-messages="RubricForm.abbr.$error" ng-show="RubricForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <bh-account-select
+      id="third_party_account_id"
+      account-id="RubricModalCtrl.rubric.third_party_account_id"
+      label="FORM.LABELS.FOUR_ACCOUNT"
+      classe="4"
+      name="third_party_account_id"
+      on-select-callback="RubricModalCtrl.selectAccountFour(account)"
+      exclude-title-accounts="true"
+      required="true"
+      validation-trigger="RubricForm.$submitted">
+    </bh-account-select>
+
+    <bh-account-select
+      id="costs_account_id"
```

Comment:
A better name is "Expense Account".  Tax is an expense, for example.  The account type it will use is an expense account as well.